### ISSUE TITLE
Cannot upgrade to 5.16

### ISSUE THREAD

**Issue by derekcentrico** [pi-hole/pi-hole#5222]
I'm trying to upgrade both Debian Buster and Ubuntu 22.04 VMs I have here and at my parents.  Have had zero problems upgrading for the four or so years using Pihole.  Suddenly this message:

```
  [i] Performing unattended setup, no dialogs will be displayed
/etc/.pihole/automated install/basic-install.sh: line 2616: getVal: command not found

  Unable to complete update, please contact Pi-hole Support
```

No clue what else to give you on this.

**Comment by yubiuser** [comment]
We are aware of the issue and working on a fix.

**Comment by THETCR** [comment]
The function `getVal` is not declared in `/opt/pihole/utils.sh`.
Wouldn't it be better to run `clone_or_update_repos` first and then source `utils.sh` from the repo?

**Comment by deathbybandaid** [comment]
Good to see I'm not alone!

**Comment by dschaper** [comment]
Jeeze, the release was out for less than 5 minutes. Do you all have neural implants to alert you to update immediately?

**Comment by marstum** [comment]
It might be a good idea to add an information on discourse, to make all, who try to update, aware that you are already working on it.

**Comment by deathbybandaid** [comment]
I follow release threads via RSS

**Comment by dschaper** [comment]
> It might be a good idea to add an information on discourse, to make all, who try to update, aware that you are already working on it.

The release was removed from the 'latest' tag so no one should be getting it now.

**Comment by marstum** [comment]
> Jeeze, the release was out for less than 5 minutes. Do you all have neural implants to alert you to update immediately?

There was fragrance in the air... :-)

**Comment by PromoFaux** [comment]
We haven't even announced the release yet ;)

**Comment by yubiuser** [comment]
The issue should be fixed with https://github.com/pi-hole/pi-hole/pull/5226

**Comment by mrcblt** [comment]
Upgrade works as expected now, thanks a lot!

**Comment by r4m3u5** [comment]
> The issue should be fixed with #5226

Just tested. Works fine now.
Many thanks for the quick fix!

**Comment by mad-tunes** [comment]
Got a notification about a new release and tried it within a few minutes
Had this problem so rolled my ubuntu VM back. Tried again and got the same

Found this thread and it was was already sorted
'pihole -up' again and update applied just fine
That might be the quickest I've ever seen, thanks & nice work!
